File: PythonProject/Week 2/models/stream.py
class Stream:
    def __init__(self, stream_id, stream_name, dept_id, duration_years, fees_per_semester):
        self.stream_id = stream_id
        self.stream_name = stream_name
        self.dept_id = int(dept_id)
        self.duration_years = int(duration_years)
        self.fees_per_semester = float(fees_per_semester)

    def __repr__(self):
        return f"Stream({self.stream_id}, {self.stream_name})"

File: Java/Week2/UniSystem/src/models/Stream.java
package models;



public class Stream
{
    public String streamId, streamName, deptId;

    public Stream(String input_line)
    {
        String[] p = input_line.split(",");
        streamId = p[0].trim(); streamName = p[1].trim(); deptId = p[2].trim();
    }
}
